src/onnx/parse_resize.cpp

```diff
@@ -33,49 +33,28 @@ namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace onnx {
 
-static std::vector<int>
-calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& vvv_ind,
-                     int i_dim,
-                     std::vector<std::vector<std::size_t>> vec_dims,
-                     const shape& in_s)
+static void calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& vvv_ind,
```

Comment:
- This function doesn't have to take `vec_ind` as an input parameter. It can simply be a return value, as before -- just define it at the beginning of this function.
- built-ins like n_dims and out_elements should not be const references. Just use the value please.
- Perhaps `n_dims` can be deduce here?

Reply:
Are you saying just get the n_dims and out_elements from:
vvv_ind.size() and vvv_ind[0][0].size()?